CHANGELOG.md

```diff
@@ -79,6 +79,8 @@
 
 - The PDK is no longer versioned
   [#8585](https://github.com/Kong/kong/pull/8585)
+- `pdk.response.set_header()`, `pdk.response.set_headers()`, `pdk.response.exit()` now ignore and emit warnings for mannually set `Transfer-Encoding` headers.
```

Comment:
I think this is not a breaking change since setting that header is unreliable in Openresty. I would consider this a Fix instead. Also, typo:

```suggestion
- `pdk.response.set_header()`, `pdk.response.set_headers()`, `pdk.response.exit()` now ignore and emit warnings for manually set `Transfer-Encoding` headers.
```